src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/version-trends-compare-user-adoption-metrics-performance.mdx

```diff
@@ -11,65 +11,42 @@ redirects:
-
-You can view version trend data in the mobile monitoring UI. You can also enable [email notifications](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/review-version-trends-email-reports).
+You can view version trend data in the mobile monitoring UI. Here you can compare performance across the most recent versions of your mobile app using the version trends report. On this single page you have access to the key data points you need to analyze the impact of improvements, fixes, and degradations for each version. This report provides metrics to compare usage, adoption, and key technical indicators.
 
 ## Requirements
 
-If you see this message in the **Version trends** UI: "No adoption data is available for this version because it is running an older agent," this indicates the mobile app wasn't using the New Relic mobile agent version that collects install/upgrade count data.
-
 To collect version trend information for your mobile app, make sure your agent version is:
 
-* New Relic mobile agent for [Android version 5.3.3](/docs/release-notes/mobile-release-notes/android-release-notes/) or higher
-* New Relic mobile agent for [iOS version 5.3.4](/docs/release-notes/mobile-release-notes/ios-release-notes/) or higher
-
-## View version trend data in UI [#view-version-trends]
-
-To view the **Version trends** report:
+* New Relic mobile agent for [Android agent v6.5.0](/docs/release-notes/mobile-release-notes/android-release-notes/) or higher
+* New Relic mobile agent for [iOS agent v7.0.0](/docs/release-notes/mobile-release-notes/ios-release-notes/) or higher
 
-1. Go to **[one.newrelic.com](https://one.newrelic.com) > Mobile > (select an app) > App > Version trends**.
-2. Select the **Launch** time period to compare versions by day (default), week, or month, or view all available data.
+## View version trends [#view-version-trends]
 
-   ![Version Trends iOS 2016-01-07 Launch buttons.png](./images/Version-Trends-iOS-2016-01-07-Launch-buttons_0.png "Version Trends iOS 2016-01-07 Launch buttons.png")
-3. Review the [version trend details](#details) to analyze performance, user adoption, and key technical indicators. For additional suggestions about how to interpret the data, review the [examples](#examples).
-4. Optional: To configure email settings, see [Email reports](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-monitoring-email-notifications#version-trends).
+To view the version trends report go to **[one.newrelic.com](https://one.newrelic.com) > Mobile > (select an app) > App > Version trends**.
 
-## View version trend details [#details]
+![This is a screenshot of the version trends UI](./images/version-trends-ui.png "Mobile version trends UI")
```

Comment:
I'd expand a little bit on the screenshot alt text. I like to think about what would be useful for someone who can't see the image (either because it's broken or because it's someone who can't see)

For example, something like: "A screenshot of the version trends UI that shows several app versions and the performance differences as percentages between them."